include/mbedtls/config.h

```diff
@@ -1260,20 +1260,20 @@
  */
 //#define MBEDTLS_ENTROPY_NV_SEED
 
-/* MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER
+/* MBEDTLS_PSA_CRYPTO_KEY_EXTENDED_ID
```

Comment:
What I am quite keen to change is to remove `file` in MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER and `psa_key_file_id_t`. I gave the main reason in the commit message of the commit "psa: Rename psa_key_file_id_t to psa_key_extended_id_t":
    
With PSA crypto v1.0.0, a volatile key identifier may contain a owner identifier but no file is associated to it. Thus rename the type psa_key_file_id_t to psa_key_extended_id_t to avoid a direct link with a file when a key identifier involves an owner
identifier. Rename also some related symbols.

If we stick to the goal of removing `file`, it could be MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER instead of `MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER`, and `psa_key_id_encodes_owner_t` instead of `psa_key_file_id_t`. What do you think?

Reply:
I'm ok with `MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER`. To be revisited when we decide on a name for `struct {psa_key_owner_id_t; psa_key_id_t;}`.